habana/ubi8-python-3.8/Pipfile

```diff
@@ -0,0 +1,59 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+
+[packages]
+# TensorFlow and useful extensions
+
+habana-tensorflow = "==1.9.0.580.1"
```

Comment:
It seems like we should consider replacing the explicit `==` with `~=` in most of the packages.

Reply:
these versions, don't follow the same x.y.z format
the ~= would not help in this case.

Reply:
That is true in order for the package specifier to work we might need to ignore the last part, but I am not sure if we have to do that. However,  some other packages in the pipfile have `==`  ex. from the L15 to L32

Reply:
> I am not sure if we have to do that.
Can you elaborate what you mean by this?

the package from habana package version have set version, they don't do patch updates.
include ~= would cause more problem .

>However, some other packages in the pipfile have == ex. from the L15 to L32
As stated these packages are dependency of habana-pytorch , changing there configuration will break that package which is installed from wheel file.

Reply:
Thank you for the explanation! It works for me :slightly_smiling_face: